src/anbox/cmds/session_manager.cpp

```diff
@@ -112,6 +112,9 @@ anbox::cmds::SessionManager::SessionManager()
   flag(cli::make_flag(cli::Name{"software-rendering"},
                       cli::Description{"Use software rendering instead of hardware accelerated GL rendering"},
                       use_software_rendering_));
+  flag(cli::make_flag(cli::Name{"no-touch-emulation"},
```

Comment:
We should add a snap config item for this so users can easily run a `snap set anbox touch-emulation.enable=false`

Reply:
ok I don't know very well the snap package system, but I think that you mean something like what we do with the software-rendering.enable, that in snap-wrapper.sh is evaluated and then export a variable that is evaluated in the session-manager.cpp.
so ok adding something like this seem simple and can be useful

Reply:
Correct, thanks for adding it